src/main/java/com/google/sps/data/Activity.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+public class Activity {
+  private String name;
+  private String type;
```

Comment:
Consider defining and using an enum for type because you are not accepting arbitrary types.

Reply:
I implemented it before but I thought it wasn't the right use case. If you think it is then I will.

Reply:
Fixed it.